docs/source/trainer/checkpointing.rst

```diff
@@ -129,17 +128,17 @@ will continue training from where the checkpoint left off.
     )
     trainer.fit()
 
-The above code will load the checkpoint from epoch 25, and continue training
+The above code will load the checkpoint from epoch 25 (if it exists, otherwise epoch 1) and continue training
```

Comment:
this shuold be reverted -- the code logic in the setup is just to mock things so the test passes

Reply:
got it